ethsigner/core/src/main/java/tech/pegasys/ethsigner/core/signing/filebased/FileBasedTransactionSigner.java

```diff
@@ -0,0 +1,63 @@
+/*
```

Comment:
I'd have said this is a factory - not the literal signer - i.e. the responsibilities of this class are to create a Signer - not _be_ a signer.

Reply:
Same goes for the HashicorpVault version

Reply:
That is where we started it all ... The team did not like my Helpers. They are both CredentialTransactionSigners with the logic on how to create the credentials being in the sub-classes.